server/src/runtime.ts

```diff
@@ -114,7 +126,7 @@ export class ScryptedRuntime extends PluginHttp<HttpPluginData> {
             this.shellHandler(req, res);
         });
 
-        this.shellio.on('connection', connection => {
+        this.shellIO.on('connection', connection => {
```

Comment:
For the sake of a minimal diff, leave this casing be. We'll be removing this endpoint anyways.